packages/bpk-docs/src/pages/PopoversPage/PopoversPage.js

```diff
@@ -62,27 +62,31 @@ class PopoverContainer extends Component {
     /* eslint-disable react/prop-types */
     const { input, ...rest } = this.props;
     return (
-      <BpkPopover
-        id="my-popover"
-        target={
-          input ?
-            <EnhancedInput
-              id="input"
-              name="input"
-              value="Open"
-              isOpen={this.state.isOpen}
-              onOpen={this.openPopover}
-              onChange={() => null}
-            />
-          :
-            <BpkButton onClick={this.openPopover}>Open</BpkButton>
-        }
-        onClose={this.closePopover}
-        isOpen={this.state.isOpen}
-        label="My popover"
-        closeButtonText="Close"
-        {...rest}
-      />
+      <div>
+        <BpkPopover
+          id="my-popover"
+          target={
+            input ?
+              <EnhancedInput
+                id="input"
+                name="input"
+                value="Open"
+                isOpen={this.state.isOpen}
+                onOpen={this.openPopover}
+                onChange={() => null}
+              />
+              :
+              <BpkButton onClick={this.openPopover}>Open</BpkButton>
+          }
+          onClose={this.closePopover}
+          isOpen={this.state.isOpen}
+          label="My popover"
+          closeButtonText="Close"
+          renderTarget={this.theamablePopoverTarget}
+          {...rest}
+        />
+        <div ref={(target) => { this.theamablePopoverTarget = target; }} />
```

Comment:
Have you tested the implications of this? i.e. the popover is no longer appended to the body thereforr zindex issues may occur.

Reply:
I have tested a few scenarios and it's all holding up 🤞